docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -64,56 +51,133 @@ use_pkce = true
 auth_style =
 ```
 
-Set `api_url` to the resource that returns [OpenID UserInfo](https://connect2id.com/products/server/docs/api/userinfo) compatible information.
+## Configuring your OAuth2 app
```

Comment:
This is a task and should therefore make use of numbered steps.

Reply:
Maybe it would make more sense to have a separate section formatted as a task with numbered steps? Basically, there are two main things that the user has to do - 1) create and configure an OAuth app 2) configure Grafana referencing the OAuth app that they've created. The steps under these "tasks" are pretty simple, but there are a looooot of different optional config options. What format would you suggest?